### PR TITLE
Add a new eligible banks operation

### PR DESCRIPTION
**Pull request by k-irishina**
For the record, I don't really like how the response is provided,  but it serves the purpose.

### CODE REVIEW

File: openapi/integrator/token-requestor/bankaxept.yaml [hunk 1]
```diff
@@ -63,6 +63,29 @@ paths:
         '503':
           $ref: '../components.yaml#/components/responses/503Error'
 
+  /v1/eligible-banks/{bankIdentifiers}:
+    get:
+      summary: Eligible bank identifiers
+      description: From the input of comma-separated list of bank identifiers, return the ones that are eligible for enrolment with ePayment.
+      operationId: supportedBankRegistrationNumbers
```
Comment (thxmasj) [inline openapi/integrator/token-requestor/bankaxept.yaml:70]: Naming of operationId could be consistent with the path, so `eligibleBanks`?

File: openapi/integrator/token-requestor/bankaxept.yaml [hunk 2]
```diff
@@ -63,6 +63,29 @@ paths:
...
+      parameters:
+        - name: bankIdentifiers
+          description: Comma-separated list of 4-digit bank identifiers to be checked for eligibility
+          in: path
+          schema:
+            type: string
```
Comment (thxmasj) [inline openapi/integrator/token-requestor/bankaxept.yaml:76]: Could use a `pattern` that restricts to four digit words comma separated.

File: openapi/integrator/token-requestor/bankaxept.yaml [hunk 3]
```diff
@@ -63,6 +63,29 @@ paths:
...
+          example: 9998, 2030
```
Comment (thxmasj) [inline openapi/integrator/token-requestor/bankaxept.yaml:77]: Maybe not allow space.

**Review by thxmasj — CHANGES_REQUESTED** [review]

File: openapi/integrator/token-requestor/bankaxept.yaml [hunk 4]
```diff
@@ -63,6 +63,34 @@ paths:
         '503':
           $ref: '../components.yaml#/components/responses/503Error'
 
+  /v1/eligible-banks:
+    get:
+      summary: Eligible bank identifiers
```
Comment (thxmasj) [inline openapi/integrator/token-requestor/bankaxept.yaml:68]: It is the banks that are eligible for enrolment with ePayment, not the identifiers, so I would prefer the summary `Eligible banks`.

File: openapi/integrator/token-requestor/bankaxept.yaml [hunk 5]
```diff
@@ -63,6 +63,34 @@ paths:
...
+      description: From the input of comma-separated list of bank identifiers, return the ones that are eligible for enrolment with ePayment.
```
Comment (thxmasj) [inline openapi/integrator/token-requestor/bankaxept.yaml:69]: The format of the parameter should not be repeated in the description, that is redundant and poorly maintainable. How about something like `Returns the subset of the inputted list of banks that are eligible for enrolment.` instead?

File: openapi/integrator/token-requestor/bankaxept.yaml [hunk 6]
```diff
@@ -63,6 +63,34 @@ paths:
...
+          required: true
+      responses:
+        '200':
+          description: Request processed
+          content:
+            application/json;charset=UTF-8:
+              schema:
+                type: string
+                description: Comma-separated list of eligible bank identifiers
+                example: ["9998", "0001"]
```
Comment (thxmasj) [inline openapi/integrator/token-requestor/bankaxept.yaml:90]: As the content-type is application/json, won't this actually end up as a list embedded in a json string, ie. `"[\"9998\", \"0001\"]"`?

**Review by thxmasj — CHANGES_REQUESTED** [review]

File: openapi/integrator/token-requestor/bankaxept.yaml (hunk 5 above)
Comment (k-irishina) [reply openapi/integrator/token-requestor/bankaxept.yaml:69]: I don't see why not, but sure, and I'll use slightly better English. :D

**Comment by k-irishina** [comment]
image

**Review by thxmasj — APPROVED** [review]

File: openapi/integrator/token-requestor/bankaxept.yaml (hunk 6 above)
Comment (k-irishina) [reply openapi/integrator/token-requestor/bankaxept.yaml:90]: See request/response below. Response body was created with ObjectMapper.writeValueAsString() on the list and this is the output. 
For other reviewers, there is a bug with returning a list of strings as the response body in the current openapi-generator version, as it returns then a flux of strings (resulting in horror Mono<ResponseEntity<Flux<String>>>>), so we return a string that is internally a list.

File: openapi/integrator/token-requestor/bankaxept.yaml [hunk 7]
```diff
@@ -63,6 +63,34 @@ paths:
         '503':
           $ref: '../components.yaml#/components/responses/503Error'
 
+  /v1/eligible-banks:
+    get:
+      summary: Eligible bank identifiers
+      description: From the input of comma-separated list of bank identifiers, return the ones that are eligible for enrolment with ePayment.
+      operationId: eligibleBanks
+      parameters:
+        - name: bankIdentifier
```
[the remark this answers is not in the document]
Comment (k-irishina) [reply openapi/integrator/token-requestor/bankaxept.yaml:72]: Like discussed, I think query params in singular is a better idea.

**Review by thxmasj — APPROVED** [review]

**Review by thxmasj — APPROVED** [review]

**Pull request merged**